fem/libceed/ceed.hpp

```diff
@@ -183,14 +209,20 @@ const std::string &GetCeedPath();
 void CeedPAAssemble(const CeedPAOperator& op,
                     CeedData& ceedData);
 
+/** This function initializes an arbitrary linear operator using a fully matrix
+    free decomposition. The operator details are described by the
+    struct CEEDPAOperator input. */
+void CeedMFAssemble(const CeedMFOperator& op,
+                    CeedData& ceedData);
+
 /** @brief Function that applies a libCEED PA operator. */
-void CeedAddMultPA(const CeedData *ceedDataPtr,
-                   const Vector &x,
-                   Vector &y);
+void CeedAddMult(const CeedData *ceedDataPtr,
+                 const Vector &x,
+                 Vector &y);
```

Comment:
You removed "PA" from the names of the functions, which I guess is because you have a more general behavior now. If this is the case, you should remove the "PA" from the comments.

Reply:
done.